pkg/stores/sqlproxy/proxy_store.go

```diff
@@ -85,6 +86,7 @@ var (
 			{"spec", "nodeName"}},
 		gvkKey("", "v1", "Service"): {
 			{"spec", "clusterIP"},
+			{"spec", "targetPort"},
```

Comment:
```suggestion
```

This could be a leftover from an earlier rebase - the field got removed by https://github.com/rancher/steve/pull/433

Reply:
Looks like the field was preserved during a rebase-commit that I resolved incorrectly by keeping that field.